src/ceramic.rs

```diff
@@ -22,3 +28,56 @@ async fn signer() -> JwkSigner {
     .await
     .unwrap()
 }
+
+#[derive(Deserialize, JsonSchema, Serialize)]
+#[schemars(rename_all = "camelCase", deny_unknown_fields)]
+pub struct SmallModel {
+    pub creator: String,
+    pub radius: i32,
+    pub red: i32,
+    pub green: i32,
+    pub blue: i32,
+}
+
+// Add to cpk did generation and signing. THis way I can understand it well enough to debug current
```

Comment:
Did you mean to leave this comment here for the future?

Reply:
I did add it to cpk :) https://github.com/nathanielc/ceramic-pocket-knife/pull/5